docker-entrypoint.sh

```diff
@@ -17,6 +17,19 @@ database_setup() {
   echo "Database setup complete"
 }
 
+database_migration() {
```

Comment:
Please add a comment above this function explaining what it is, and how it works vs. `database_setup`.  It might be good to link to the Prisma docs in both cases: https://www.prisma.io/docs/concepts/components/prisma-migrate/mental-model